OrbitCore/SamplingProfiler.cpp

```diff
@@ -144,9 +144,6 @@ void SamplingProfiler::AddCallStack(CallStack& a_CallStack) {
 
 //-----------------------------------------------------------------------------
 void SamplingProfiler::AddHashedCallStack(CallstackEvent& a_CallStack) {
-  if (m_State != Sampling) {
-    PRINT("Error: Callstacks can only be added while sampling.\n");
-  }
```

Comment:
should this be ignored if we are no sampling?

Reply:
No, because when sampling is stopped there are still samples that fell inside the tracing window that haven't been processed. Sure, there might be samples that have actually been collected after sampling was stopped, but they don't cause issues and this doesn't seem the place to check for them.